app/controllers/msa_components_controller.rb

```diff
@@ -4,20 +4,32 @@ def index
   end
 
   def new
-    @component = NewMsaComponentEvent.new
+    begin
+      authorize NewMsaComponentEvent
```

Comment:
this should not be part of this PR/ticket... and this logic would ideally be part of the `before_` callback instead of polluting the method code itself...